bindings/experimental/distrdf/python/DistRDF/Backends/Base.py

```diff
@@ -218,8 +218,7 @@ def mapper(current_range):
             # rdf_range = rdf.Range(current_range.start, current_range.end)
 
             # Output of the callable
-            resultptr_list = computation_graph_callable(
-                rdf, rdf_range=current_range)
+            resultptr_list = computation_graph_callable(rdf, current_range.id, rdf_range=current_range)
```

Comment:
You don't need to pass `current_range.id` as parameter if you are already passing `current_range`, right?

Reply:
Exactly! But that will change in the next PR where I will remove `current_range` completely. I prefer passing a single integer rather than the whole object